tests/unit/components/playback/controls/ButtonWidget.spec.js

```diff
@@ -0,0 +1,271 @@
+import { mount } from "@vue/test-utils";
```

Comment:
It doesn't look like there is a test for if the `is_enabled` prop gets updated after mounting to determine if that propogates all the desired changes throughout the component.  If we have a key prop we know will be frequently updated by the parent, it's good for us to confirm that functionality.

Reply:
Sure, this is required and will add one more testcase